tests/osaka/eip7692_eof_v1/eip3540_eof_v1/test_all_opcodes_in_container.py

```diff
@@ -382,3 +402,55 @@ def test_all_opcodes_stack_overflow(
         data=eof_code,
         expect_exception=exception,
     )
+
+
+@pytest.mark.parametrize(
+    "opcode",
+    sorted(data_portion_opcodes),
+)
+@pytest.mark.parametrize(
+    "trunc_all",
+    [False, True],
+)
+@pytest.mark.parametrize(
+    "compute_max_stack_height",
+    [False, True],
+)
+def test_truncated_data_portion_opcodes(
+    eof_test: EOFTestFiller,
+    opcode: Opcode,
+    trunc_all: bool,
+    compute_max_stack_height: bool,
+):
+    """
+    Test that an instruction with data portion and truncated immediate bytes
+    (therefore a terminating instruction is also missing) invalidates EOF.
+    """
+    opcode_with_data_portion: bytes = bytes(opcode[1])
+
+    if len(opcode_with_data_portion) == 2 and trunc_all:
+        pytest.skip("can only be truncated one-way")
```

Comment:
I don't think the use of `pytest.skip` is good here. We want to exclude some combination of parameters but don't want to print skip logs for these tests.

```
tests/osaka/eip7692_eof_v1/eip3540_eof_v1/test_all_opcodes_in_container.py::test_truncated_data_portion_opcodes[fork_Osaka-eof_test-compute_max_stack_height_True-trunc_all_True-opcode_CALLF] SKIPPED (max_stack_height is 0)                                         [916/922]
tests/osaka/eip7692_eof_v1/eip3540_eof_v1/test_all_opcodes_in_container.py::test_truncated_data_portion_opcodes[fork_Osaka-eof_test-compute_max_stack_height_True-trunc_all_True-opcode_JUMPF] SKIPPED (max_stack_height is 0)                                         [917/922]
tests/osaka/eip7692_eof_v1/eip3540_eof_v1/test_all_opcodes_in_container.py::test_truncated_data_portion_opcodes[fork_Osaka-eof_test-compute_max_stack_height_True-trunc_all_True-opcode_DUPN] SKIPPED (can only be truncated one-way)                                  [918/922]
tests/osaka/eip7692_eof_v1/eip3540_eof_v1/test_all_opcodes_in_container.py::test_truncated_data_portion_opcodes[fork_Osaka-eof_test-compute_max_stack_height_True-trunc_all_True-opcode_SWAPN] SKIPPED (can only be truncated one-way)                                 [919/922]
tests/osaka/eip7692_eof_v1/eip3540_eof_v1/test_all_opcodes_in_container.py::test_truncated_data_portion_opcodes[fork_Osaka-eof_test-compute_max_stack_height_True-trunc_all_True-opcode_EXCHANGE] SKIPPED (can only be truncated one-way)                              [920/922]
tests/osaka/eip7692_eof_v1/eip3540_eof_v1/test_all_opcodes_in_container.py::test_truncated_data_portion_opcodes[fork_Osaka-eof_test-compute_max_stack_height_True-trunc_all_True-opcode_EOFCREATE] SKIPPED (can only be truncated one-way)                             [921/922]
tests/osaka/eip7692_eof_v1/eip3540_eof_v1/test_all_opcodes_in_container.py::test_truncated_data_portion_opcodes[fork_Osaka-eof_test-compute_max_stack_height_True-trunc_all_True-opcode_RETURNCONTRACT] SKIPPED (can only be truncated one-way)                        [922/922]
```

Reply:
Please see https://github.com/ipsilon/execution-spec-tests/pull/5